core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -40,6 +40,25 @@ oppia.directive('simpleEditorSidebar', [
           $scope.questionList = SimpleEditorManagerService.getQuestionList();
           $scope.ID_PREFIX = QuestionIdService.SIDEBAR_PREFIX;
           $scope.sidebarModeService = SimpleEditorSidebarModeService;
+          $scope.addNewQuestion = function() {
+            if (!SimpleEditorManagerService.canAddNewQuestion()) {
+              return;
+            }
+            $scope.sidebarModeService.setModeToReadonly();
+            SimpleEditorManagerService.addNewQuestion();
+            $scope.questionList = SimpleEditorManagerService.getQuestionList();
+            var end = $scope.questionList.getQuestionAt(
+              $scope.questionList.getQuestionCount() - 1);
+        // for (var i = 0; i < $scope.questionList.getQuestionCount(); i++) {
+              // console.log($scope.questionList.getQuestionAt(1));
+            // }
+//         console.log(JSON.stringify($scope.questionList.getLastQuestion()));
+//            console.log(JSON.stringify(
+//SimpleEditorManagerService.getQuestionList().getLastQuestion()));
+            $scope.scrollToQuestion(end);
```

Comment:
@saeedjassani At this point, do the following instead:

```
    $timeout(function() {
      $scope.scrollToQuestion(end);
    }, 0);
```

This will move the scroll action to the bottom of the JS execution queue, which allows the DOM to stabilize before the scroll is performed. A bit inelegant, but that should solve it for now. Note that you'll need to add $timeout to the list of dependencies.